AuRoRA/src/scs/scs/cnc.py

```
@@ -38,303 +23,139 @@
-                "/v1/chat/completions",
-                json=payload,
-            ) as resp:
-
-                if resp.status_code != 200:
-                    err = f"vLLM HTTP {resp.status_code}"
-                    self.get_logger().error(f"❌ {err}")
-                    self._publish({"type": "error", "content": err})
-                    return ""
-
+            async with self._http.stream("POST", "/v1/chat/completions", json=payload) as resp:
+                if resp.status_code != 200: return ""
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)